sdk/batch/Microsoft.Azure.Management.Batch/tests/ScenarioTests/PoolTests.ScenarioTests.cs

```diff
@@ -118,8 +119,8 @@ public async Task BatchPoolEndToEndAsync()
                     // Verify get operation
                     var pool = await this.BatchManagementClient.Pool.GetAsync(resourceGroupName, batchAccountName, iaasPoolName);
                     Assert.Equal("STANDARD_A1", pool.VmSize);
-                    Assert.Null(pool.DisplayName);
-                    Assert.Equal(AllocationState.Steady, pool.AllocationState);
+                    Assert.Equal(displayName, pool.DisplayName);
```

Comment:
That `displayName` was previously not respected was a bug. This was fixed in the latest release of the Service. It is not called out in the changelog as this bug was fixed for all REST API versions.